cmd/skupper/skupper_link.go

```diff
@@ -154,6 +180,11 @@ func NewCmdLinkStatus(newClient cobraFunc) *cobra.Command {
 					}
 				}
 			} else {
+				if verbose {
+					fmt.Println("In order to provide detailed information about the link, please specify the name")
```

Comment:
This validation could be done before calling `SiteConfigInspect`. If you do a `link status --verbose` without
specifying the link name in a remote cluster (ibmcloud, in example), it delays a bit before displaying the
error message.

Reply:
Good catch, I'll fix it.